section3/set11/exercises/11-average/main.go

```diff
@@ -60,22 +60,18 @@ func main() {
 			fmt.Println("Please tell me numbers (maximum 5)")
 			return
 		}
-
 		avg[i] = n
 	}
-	fmt.Printf("Array: %v\n", avg)
+	//fmt.Printf("Array: %v\n", avg)
 
 	sum := 0
 	// create for loop to prompt user to input values
 	for i := 0; i < len(os.Args)-1; i++ {
```

Comment:
here in comparison, take the length from avg array instead-of os.Args

Reply:
error has been fixed